test/unit/test-amp-story-player.js

```diff
@@ -69,6 +70,11 @@ describes.realWin('AmpStoryPlayer', {amp: false}, env => {
 
   beforeEach(() => {
     win = env.win;
+    oldPrototype = Object.getPrototypeOf(AmpStoryPlayer);
+    // Make the AmpStoryPlayer "native" to the 'realWin' window.
+    Object.setPrototypeOf(AmpStoryPlayer, win.HTMLElement);
+    Object.setPrototypeOf(AmpStoryPlayer.prototype, win.HTMLElement.prototype);
+    win.customElements.define('amp-story-player', AmpStoryPlayer);
```

Comment:
Could you clarify what problem this piece of code is solving?
My knowledge of how `extends` plays with prototypes is not perfect, but isn't it supposed to handle everything for you?

Reply:
Added a comment, let me know if it's still not clear.